packratt/tests/conftest.py

```diff
@@ -24,6 +25,22 @@ def test_cache(tmp_path_factory):
         set_cache(old_cache)
 
 
+content = '''\
+'/test/ms/2020-06-04/google/smallest_ms.tar.gz':
+  file_id: 1wjZoh7OAIVEjYuTmg9dLAFiLoyehnIcL
+  hash: 4d548b22331fb3cd3256b1b4f37a41cf
+  description: >
+    Small testing Measurement Set, stored on Google Drive
+'''
+
+
 @pytest.fixture(scope="session")
-def registry():
+def registry(tmp_path_factory):
+
+    user_conf_path = tmp_path_factory.mktemp("conf")
+    USER_REGISTRY = user_conf_path / "registry.yaml"
+    CONTENT = yaml.dump(content)
+    USER_REGISTRY.write_text(CONTENT)
+    assert USER_REGISTRY.read_text() == CONTENT
+
     return load_registry()
```

Comment:
This is absolutely the right idea.

The only problem is that ``packratt.registry.USER_REGISTRY`` will need to point at the test user registry for the registry loading to work.

This is probably achievable with something like this:

```python
# At the top
from unittest.mock import patch

...

with patch("packratt.registry.USER_REGISTRY", USER_REGISTRY):
    return load_registry() 
```